src/plugins/BlockingCommandPlugin.ts

```diff
@@ -146,10 +146,32 @@ export default class BlockingCommandPlugin extends AbstractPlugin {
       unblockedJids.forEach(jid => {
          if (!newList.includes(jid)) {
             getElements(jid).removeClass('jsxc-blocked');
+
+            let contact = this.pluginAPI.getContact(new JID(jid));
+            if (contact)
+            {
+                let chatwindow =  contact.getChatWindow();
+                if (chatwindow)
+                {
+                   chatwindow.getElement().removeClass('jsxc-blocked');
```

Comment:
This should not be needed, since `getElements` should include roster items and windows.

Reply:
i had the problem, that if i unblocked an user and opend the chatwindow again the title was still lined through